internal/controllers/clusterextension_controller_test.go

```diff
@@ -821,7 +821,7 @@ func TestClusterExtensionVersionNoChannel(t *testing.T) {
 	cond = apimeta.FindStatusCondition(clusterExtension.Status.Conditions, ocv1alpha1.TypeInstalled)
 
 	require.NotNil(t, cond)
-	require.Equal(t, metav1.ConditionUnknown, cond.Status)
+	require.Equal(t, metav1.ConditionFalse, cond.Status)
```

Comment:
This looks good to me! We are anyway going to change the installation conditions, so marking install as False when there is error is fine for now.